.circleci/config.yml

```diff
@@ -1,6 +1,10 @@
 defaults: &defaults
   docker:
     - image: circleci/ruby:2.4.3-browsers
+    - image: circleci/redis
+      command: redis-server --requirepass redispassword
+      environment:
+        TZ: "/usr/share/zoneinfo/Europe/Amsterdam"
```

Comment:
Kun je de port hiervan ook opgeven in het wait for db commando? (regel 65)

Reply:
done